Automattic/wp-calypso
# [PBD interim fix] Collective PR for clean thank you page

#### Changes proposed in this Pull Request

This PR is part of the PBD interim fix (https://github.com/Automattic/wp-calypso/issues/38110).

It updates checkout "thank you" page for users upgrading to Business in order to install a plugin or a theme.

**Before:**

<img width="1029" alt="Zrzut ekranu 2019-12-3 o 14 56 52" src="https://user-images.githubusercontent.com/205419/70057278-27541900-15dd-11ea-9846-b6f513422e6b.png">

**After:**

<img width="1020" alt="Zrzut ekranu 2019-12-3 o 14 56 41" src="https://user-images.githubusercontent.com/205419/70057268-21f6ce80-15dd-11ea-9532-8b441e1c4be3.png">

#### Potential issues

This will **always** say the site has been launched, even if the user already launched their site earlier. It may be a tad confusing for some users, but reliably figuring out if the site was literally just launched adds a lot of complexity (page refreshes and so on). This feels like a good trade-off. I'm curious about everyone's thoughts though - we could always adjust that.

#### Testing instructions

1. Upgrade to personal/premium/business/ecommerce from both signup and /plans, confirm you see the **Before** page
1. Create a free/personal/premium site
1. Go to /plugins and click on the business plan upgrade nudge (both on plugins list and specific plugin's page)
1. Upgrade to business
1. Confirm the thank you page looks like **After** screenshot above
1. Repeat 2-5 from /themes
